playbooks/roles/cyclecloud_cluster/projects/common/cluster-init/scripts/5-default.sh.j2

```diff
@@ -29,3 +29,6 @@ esac
 if [ ! -d /usr/share/Modules ]; then
```

Comment:
This folder currently exists on the HPC CentOS image, i.e. the symlink is not created.
Not quite sure what purpose this serves